public/app/types/dashboard.ts

```diff
@@ -38,6 +38,7 @@ export interface DashboardMeta {
 }
 
 export interface DashboardDataDTO {
+  [x: string]: any;
```

Comment:
This looks a bit broad. Why is this needed?

Reply:
It's definitely broader than it needs to be, but the original type definition doesn't really reflect the structure it's supposed to either. I guess something more accurate would be like:
```ts
{
  annotations: { list: any[] };
  editable: any;
  gnetId: any;
  graphTooltip: number;
  id: number;
  links: any;
  panels: PanelModel[];
  schemaVersion: number;
  style: 'light' | 'dark';
  tags: any;
  templating: { list: any[] };
  time: any;
  timepicker: {
    collapse?: boolean,
    enable?: boolean,
    hidden?: boolean,
    notice?: boolean,
    now?: boolean,
    refresh_intervals?: string[],
    status?: string,
    time_options?: string[],
    type?: string,
  };
  timezone: any;
  title: string;
  uid: string;
  version: number;
}
```

though it's hard to know if this is a complete definition since documentation regarding the API seems to be outdated